frontend/src/api/DataPipeline/updateDataPipelineEnvironment.js

```diff
@@ -0,0 +1,24 @@
+import { gql } from 'apollo-boost';
+
+const updateDataPipelineEnvironment = ({ input }) => ({
+  variables: {
+    input
+  },
+  mutation: gql`
+    mutation updateDataPipelineEnvironment($input: UpdateDataPipelineEnvironmentInput) {
```

Comment:
Here we would also need to change the input if we were to use the same one as the "NewDataP..Input"